packages/visual-dev/src/components/Badge/Badge.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
+import styled from 'styled-components'
+import { Icon } from "../Icons";
+
+import * as Styles from './Styles'
+
+interface BadgeProps {
+    status: "info" | "success" | "active" | "critical" | "warning";
+    children?: any[] | string;
+    icon?: string;
+}
+
+const BadgeStyled = styled.a<BadgeProps>`
```

Comment:
Not sure it should be an `a` tag, probably `div` or even `span` instead? `a` tags should be used mainly for links to other pages